scripts/truffle/bot.js

```diff
@@ -123,13 +128,39 @@ async function processBite(cdp) {
   console.log("### WITHDRAWN ###: " + radToDAI(rad) + " DAI");
 }
 
+// Ensure that the MEMBER has expected DAI balance before topup
+async function ensureDAIBalance(cdp, neededRadBal, opt) {
```

Comment:
need to make sure user always have 1 dai more than what needed, to avoid rounding errors.